### DISCUSSION TITLE
Is download guide outdated?

### DISCUSSION THREAD

**Discussion by 1centu** [blizzhackers/documentation#19] [Ideas]
Can't download the bot. Installed tortoiseSVN but when i add this link https://github.com/blizzhackers/kolbot/trunk and hit ok nothing happens. It says unable to connect to repository at URL.

**Comment by ejt1** [comment]
Using TortoiseSVN is deprecated and should not be used!

The documentation was updated a while a go with a [guide to use Github Desktop](https://bhdocs.github.io/d2bot/download#using-github-desktop)

There is also a guide on [how to install using ZIP](https://bhdocs.github.io/d2bot/download#as-zip)

**Reply by 1centu** [reply]
Thanks a lot mate. I was using a very old version of it and wanted to see the new changes.
